lib/WebpackConfig.js

```diff
@@ -456,6 +452,10 @@ class WebpackConfig {
             throw new Error('Argument 1 to configureBabelPresetEnv() must be a callback function.');
         }
 
+        if (this.doesBabelRcFileExist()) {
+            logger.warning('The "callback" argument of configureBabelPresetEnv() will not be used because your app already provides an external Babel configuration (a ".babelrc" file, ".babelrc.js" file or "babel" key in "package.json").');
```

Comment:
as that's the only argument, the whole call is useless. Maybe we should make it an error then.

Reply:
👍